Jenkinsfile

```diff
@@ -626,6 +626,108 @@ pipeline {
                     }
                 }
 
+                stage('MIOpenTensile Half Hip Release') {
```

Comment:
If these tests are really "Full", then the Jenkins testing time will increase significantly. I am sure that we cannot afford it. CI is already a bottleneck in the development process (and terribly irritating).

Reply:
They will not be test in "Full". 
If you check test/Cmakelist.txt, you will find when MIOPEN_USE_MIOPENTENSILE=ON, the majority of the tests will be suppressed except gemm related tests.